src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -168,6 +168,10 @@ public Lint ()
 			// for the MonoPackageManager.java since we have to use a static to keep track of the
 			// application instance.
 			{ "StaticFieldLeak", new Version(26, 0, 2) },
+			// We need to hard code this test as disabled in because Lint will issue a error
```

Comment:
This comment reads a bit wonky; are you missing a word between "in" and "because"?